src/component/variantPage/FunctionalPrediction.tsx

```diff
@@ -52,8 +55,24 @@ class FunctionalPrediction extends React.Component<IFunctionalPredictionProps> {
             genomeNexusData &&
             genomeNexusData.transcript_consequences &&
             genomeNexusData.transcript_consequences[0].polyphen_prediction;
+        const amClass =
+            genomeNexusData &&
+            genomeNexusData.annotation_summary &&
+            genomeNexusData.annotation_summary.transcriptConsequenceSummary &&
+            genomeNexusData.annotation_summary.transcriptConsequenceSummary.alphaMissense
+                ? genomeNexusData.annotation_summary.transcriptConsequenceSummary.alphaMissense.pathogenicity
+                : 'N/A';
```

Comment:
```suggestion
        const amClass = genomeNexusData?.annotation_summary?.transcriptConsequenceSummary?.alphaMissense?.pathogenicity || undefined;

```
Same for amPathogenicityScore

Reply:
changed it